dev/tests/api-functional/testsuite/Magento/GraphQl/Catalog/CategoryTest.php

```diff
@@ -112,6 +112,21 @@ public function testCategoriesTree()
         );
     }
 
+    public function testNonExistentCategoryWithProductCount()
+    {
+        $query = <<<QUERY
+{
+  category(id: 99) {
+      product_count
+    }
+}
+QUERY;
+
+        $response = $this->graphQlQuery($query);
+        $expectedResponse = ['category' => null];
```

Comment:
It will be better to back some error like `Category doesn't exist`

Example for CMS page
https://github.com/magento/graphql-ce/blob/2.3-develop/app/code/Magento/CmsGraphQl/Model/Resolver/Page.php#L77

Thanks

Reply:
@naydav that's done.